src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -1,11 +1,12 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
 /**
  * Parameterize this service and add its implementation in a separate class.
  */
-public interface MachineService {
+public interface MachineService<T> {
```

Comment:
if we use only `T` type, we can parameterize this interface with any type of class (It can be Person, String, Object...) Is it valid behavior for MachineService interface?
@gigbat be attentive))

Reply:
Okey

Reply:
You are right, thx)